.circleci/config.yml

```diff
@@ -40,7 +45,36 @@ jobs:
 
       - run: npm install
       - run: git config --global push.default simple
-      - run: npm run release
+      - run: npm run build:package
+      - run: cd ./build
+      - run: echo "//registry.npmjs.org/:_authToken=$npm_TOKEN" > ~/repo/.npmrc
+      - run: npm publish --access public
+      # build.js runs the git tagging process
+      - run: node build.js
+
+  deploy-website:
+    docker:
+      - image: circleci/node:8.11.1
+
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "package.json" }}
+          # fallback to using the latest cache if no exact match is found
+          - v1-dependencies-
+
+      - run: npm install
+      - run: git config --global push.default simple
+      - run: npm run build:storybook
+      - run: mkdir .static-website
+      - run: cd .static-website
+      - run: git clone -b gh-pages git@github.com:UvitaTeam/vivy-components .
```

Comment:
is it better to just force push to gh-pages?

Reply:
This way is a bit more work but less destructive.